static/js/settings_profile_fields.js

```diff
@@ -247,22 +241,22 @@ export function parse_field_choices_from_field_data(field_data) {
     return choices;
 }
 
-function set_up_external_account_field_edit_form(field_elem, url_pattern_val) {
-    if (field_elem.$form.find("select[name=external_acc_field_type]").val() === "custom") {
-        field_elem.$form.find("input[name=url_pattern]").val(url_pattern_val);
-        field_elem.$form.find(".custom_external_account_detail").show();
-        field_elem.$form.find("input[name=name]").val("").closest(".control-group").show();
-        field_elem.$form.find("input[name=hint]").val("").closest(".control-group").show();
+function set_up_external_account_field_edit_form($field_elem, url_pattern_val) {
+    if ($field_elem.find("select[name=external_acc_field_type]").val() === "custom") {
+        $field_elem.find("input[name=url_pattern]").val(url_pattern_val);
+        $field_elem.find(".custom_external_account_detail").show();
+        $field_elem.find("input[name=name]").val("").closest(".control-group").show();
+        $field_elem.find("input[name=hint]").val("").closest(".control-group").show();
     } else {
-        field_elem.$form.find("input[name=name]").closest(".control-group").hide();
-        field_elem.$form.find("input[name=hint]").closest(".control-group").hide();
-        field_elem.$form.find(".custom_external_account_detail").hide();
+        $field_elem.find("input[name=name]").closest(".control-group").hide();
+        $field_elem.find("input[name=hint]").closest(".control-group").hide();
```

Comment:
You have not changed `control-group` here to `input-group` and also in `set_up_create_field_form` due to which the fields are not hidden/shown correctly for `External account` type fields. Also, we can make this change in a prep commit as Tim mentioned, as margin CSS for both the classes is same and is the only thing used here.